WIP-CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+### Changed
```

Comment:
This file is not needed anymore, please rebase on main and remove it :slightly_smiling_face: